src/Models/NonhydrostaticModels/nonhydrostatic_model.jl

```diff
@@ -36,7 +36,7 @@ mutable struct NonhydrostaticModel{TS, E, A<:AbstractArchitecture, G, T, B, R, S
            velocities :: U        # Container for velocity fields `u`, `v`, and `w`
               tracers :: C        # Container for tracer fields
             pressures :: Φ        # Container for hydrostatic and nonhydrostatic pressure
-        diffusivities :: K        # Container for turbulent diffusivities
+   diffusivity_fields :: K        # Container for turbulent diffusivities
           timestepper :: TS       # Object containing timestepper fields and parameters
       pressure_solver :: S        # Pressure/Poisson solver
     immersed_boundary :: I        # Models the physics of immersed boundaries within the grid
```

Comment:
Like for the hydrostatic model, the formatting of the model properties is mangled.